src/commands/config/list.js

```diff
@@ -1,16 +1,31 @@
 // @flow
 import * as options from '../../utils/options';
 import { BoltError } from '../../utils/errors';
+import * as yarn from '../../utils/yarn';
+import { DEPENDENCY_TYPE_FLAGS_MAP } from '../../constants';
 
-export type ConfigListOptions = {};
+export type ConfigListOptions = {
+  cwd?: string
+};
 
 export function toConfigListOptions(
   args: options.Args,
   flags: options.Flags
 ): ConfigListOptions {
-  return {};
+  if (
+    args.length !== 0 ||
+    Object.keys(DEPENDENCY_TYPE_FLAGS_MAP).length !== 0
```

Comment:
what is this for?

Reply:
Oops, just found flags are valid.

Reply:
wait flags are valid but they are ignored in yarn. So can we also do the same ?